src/Blocks/components/paragraph/paragraph-style.scss

```diff
@@ -0,0 +1,50 @@
+$paragraph: (
+	colors: global-settings(colors),
+	link-color: global-settings(colors, primary),
+	sizes: (
+		mobile: (
+			default: (
+				font-size: 18px,
+				line-height: 1.56,
+			),
+			small: (
+				font-size: 14px,
+				line-height: 1,
+			),
+		),
+		large: (
+			default: (
+				font-size: 22px,
+			),
+			small: (
+				font-size: 18px,
+			),
+		),
+	),
+);
+
+.paragraph {
+	// @extend %paragraph-reset;
+	// color: map-get-deep($paragraph, colors, primary);
```

Comment:
Commented styles

Reply:
Good find, paragraph and heading don't even need to be in here, remove both.